src/main/java/ch/njol/skript/effects/EffItemLifetime.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Unlimited Item Lifetime")
+@Description("Makes a dropped item have unlimited lifetime, meaning it won't despawn from vanilla Minecraft timer.")
+@Examples({
+	"enabled unlimited lifetime of all dropped items"
+})
+@Since("INSERT VERSION")
+public class EffItemLifetime extends Effect {
+
+	static {
+		Skript.registerEffect(EffItemLifetime.class,
+			"enable unlimited lifetime (of|for) %entities%",
+			"disable unlimited lifetime (of|for) %entities%");
```

Comment:
Not a huge fan of `of`. What do you think about additional patterns like `make <dropped items> lifetime unlimited` or `make <dropped item> never despawn`